tests/scripts/thread-cert/node.py

```diff
@@ -363,6 +365,46 @@ def dns_dig(self, server: str, name: str, qtype: str):
 
         return dig_result
 
+    def call_dbus_method(self, *args):
+        args = ' '.join(args)
```

Comment:
Would cause issues if there are spaces in `args`.

Maybe not in this PR, better to make `self.bash` accept a list of string and pass args as a list.

Reply:
ACK. I think this could generally be avoided by the caller.